### PR TITLE
sensors.AFRICA embeds

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

This allows to get:
-  the sensor map using the endpoint `https://sensors.africa/embeded/air/map?city=<city>`.
example:
```html
<iframe src="https://localhost:3001/embeded/air/map?city=nairobi" height='500px' width='800px'></iframe>
```
-  the weekly data graph using the endpoint `https://sensors.africa/embeded/air/graph?city=<city>`.
example:
```html
<iframe src="https://localhost:3001/embeded/air/graph?city=nairobi" height='500px' width='800px'></iframe>
```

-  the dial component using the endpoint `https://sensors.africa/embeded/air/dial?city=<city>`.
example:
```html
<iframe src="https://localhost:3001/embeded/air/dial?city=nairobi" height='400px' width='100%'></iframe>
```

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

### Dial

Screenshot from 2019-03-20 08-00-11

Screenshot from 2019-03-20 08-01-24

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/Embeds/AirMap.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CITIES_LOCATION = {
+  nairobi: '12/-1.2709/36.8169',
+  lagos: '12/6.4552/3.4198',
+  'dar-es-salaam': '12/-6.8555/39.1518'
+};
+
+function AirMap({ location }) {
```
Comment (kilemensi) [inline src/components/Embeds/AirMap.js]: QQ The `SensorMap` itself is already an `iframe`, why don't we return just that?

File: src/components/Embeds/AirGraph.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      stats: []
+    };
+  }
+
+  componentDidMount() {
+    const { location } = this.props;
+    const params = new URLSearchParams(location.search);
+    const city = params.get('city');
+
+    const formatWeeklyP2Stats = data => {
```
Comment (kilemensi) [inline src/components/Embeds/AirGraph.js]: Lets find a way to centralize these calculations... May be move them into the `QualityStatsGraph` component and just pass data from this component and `City` page component?.

Comment (karimkawambwa) [reply src/components/Embeds/AirGraph.js]: 😢  okay

File: src/components/Embeds/AirMap.js (hunk 1 above)
Comment (karimkawambwa) [reply src/components/Embeds/AirMap.js]: sure, i don't have much resistance for that, i just found it weird to have iframe and another iframe.

Comment (kilemensi) [reply src/components/Embeds/AirMap.js]: I meant lets return the _inner_ `iframe` itself. It'll be weird having two iframes!

File: src/components/City/HostSensors/ShareButton/Embed.js [hunk 3]
```diff
@@ -18,13 +18,16 @@ const styles = () => ({
...
-            height="570px" width="800px" allowfullscreen>
+    <iframe src="https://sensors.africa/embeded/air/dial?city=${city.slug}"
+            style="border: none;"
+            name="sensors.AFRICA | ${city.name}"
+            scrolling="no"
+            frameborder="0"
+            height="400"
+            width="100%"
+            allowfullscreen>
     </iframe>`;
```
Comment (karimkawambwa) [inline src/components/City/HostSensors/ShareButton/Embed.js]: 👍

**Pull request merged**